### ISSUE TITLE
Default legend parser for menu items

### ISSUE THREAD

**Issue by allthesignals** [NYCPlanning/labs-citymap#170]
If legend configuration exists and if no block is passed to `{{menu-item}}`, we should use a default, opinionated legend renderer.

**Comment by allthesignals** [comment]
Proposed API:
```
      {{#lookup-layer-group for='zoning-districts' as |layerGroup|}}
        {{menu.menu-item
          title=layerGroup.model.title
          tooltip=layerGroup.model.titleTooltip
          active=zoning-districts}}
      {{/lookup-layer-group}}
```

**Comment by allthesignals** [comment]
Closed by d3281ab
